pyproject.toml

```
@@ -1,5 +1,153 @@
+license = "Apache-2.0"
+requires-python = ">=3.8, <4"
+authors = [
+    { name = "Ethyca, Inc.", email = "fidesteam@ethyca.com" },
+]
+keywords = [
+    "data privacy",
+    "privacy as code",
+    "data privacy compliance",
+]
+classifiers = [
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Software Development :: Libraries",
+]
+dependencies = [
+    "alembic==1.8.1",
+    "APScheduler==3.9.1",
+    "asyncpg==0.25.0",
+    "celery[pytest]==5.2.7",
+    "click==8.1.3",
+    "colorama>=0.4.3",
+    "cryptography~=3.4.8",
+    "dask==2022.8.0",
+    "deepdiff==5.8.0",
+    "emails",
+    "fastapi[all]==0.81.0",
+    "fastapi-pagination[sqlalchemy]~= 0.9.3",
+    "fideslang==1.3.0",
+    "fideslib==3.1.1",
+    "fideslog==1.2.3",
+    "GitPython==3.1.27",
+    "hvac==0.11.2",
+    "loguru>=0.5,<0.6",
+    "Jinja2==3.1.2",
+    "multidimensional_urlencode==0.0.4",
+    "openpyxl==3.0.9",
+    "pandas==1.4.3",
+    "passlib[bcrypt]==1.7.4",
+    "plotly==5.4",
+    "psycopg2-binary==2.9.3",
+    "pydantic<1.10.0",
+    "pydash==5.1.0",
+    "python-jose[cryptography]==3.3.0",
+    "PyJWT==2.4.0",
+    "pyyaml>=5,<6",
+    "sqlalchemy[asyncio]==1.4.27",
+    "sqlalchemy-stubs==0.4",
+    "SQLAlchemy-Utils==0.38.3",
+    "toml>=0.10.1",
+    "Unidecode==1.3.4",
+]
+
+[project.optional-dependencies]
+dev = [
+    "black==22.8.0",
+    "Faker==14.1.0",
+    "GitPython==3.1.27",
+    "isort==5.10.1",
+    "mypy==0.971",
+    "nox==2022.8.7",
+    "packaging==21.3",
+    "pre-commit==2.20.0",
+    "pylint==2.15.0",
+    "pytest-asyncio==0.19.0",
+    "pytest-cov==3.0.0",
+    "pytest-env==0.6.2",
+    "pytest==7.1.2",
+    "requests-mock==1.9.3",
+    "setuptools>=64.0.2",
+    "sqlalchemy-stubs",
+    "types-PyYAML==6.0.11",
+    "types-redis==4.3.4",
+    "types-requests",
+    "types-toml==0.10.8",
+    "types-ujson==5.4.0",
+    "types-urllib3==1.26.23",
+    "xenon==0.9.0",
+    "build==0.8.0",
+]
+aws = [
+    "boto3~=1.24.46",
+]
+bigquery = [
+    "sqlalchemy-bigquery==1.4.4",
+]
+mongo = [
+    "pymongo==3.12.0",
+]
+mssql = [
+    "pyodbc==4.0.34",
+]
+mysql = [
+    "PyMySQL==1.0.2",
+]
+okta = [
+    "okta==2.5.0",
+]
+redis = [
+    "redis==3.5.3",
+    "fastapi-caching[redis]",
+]
+redshift = [
+    "sqlalchemy-redshift==0.8.11",
+]
+snowflake = [
+    "snowflake-sqlalchemy==1.4.1",
+]
+all = [
+    "boto3~=1.24.46",
+    "sqlalchemy-bigquery==1.4.4",
+    "pymongo==3.12.0",
+    "PyMySQL==1.0.2",
+    "okta==2.5.0",
+    "redis==3.5.3",
+    "fastapi-caching[redis]",
+    "sqlalchemy-redshift==0.8.11",
+    "snowflake-sqlalchemy==1.4.1",
+]
+
+[project.urls]
+Homepage = "https://github.com/ethyca/fides"
+
+[tool.hatch.build.targets.sdist]
+include = [
+    "/src",
+]
+
+[tool.hatch.build.targets.wheel]
+include = [
+    "/src",
```

https://hatch.pypa.io/latest/config/build/#packages

```suggestion
packages = [
    "/src/fides",
```

---

Alright. Thanks for the heads up!